docs/support_playbook.md

```diff
@@ -336,22 +336,43 @@ application_choice = ApplicationChoice.find(APPLICATION_CHOICE_ID)
 new_course_option = CourseOption.find(NEW_COURSE_OPTION_ID)
 zendesk_url = ZENDESK_URL
 
-# confirm the deferral in a new course
-application_choice.update_course_option_and_associated_fields!(new_course_option, audit_comment: zendesk_url)
-```
-
-A **conditional offer** would move the candidate to a pending conditions state:
-
-```ruby
-# change the status to pending conditions (if it is a conditional deferred offer)
-application_choice.update!(status: 'pending_conditions', audit_comment: zendesk_url)
-```
-
-An **unconditional offer** would move the candidate to a recruited state:
-
-```ruby
-# change the status to recruited (if it is an unconditional deferred offer)
-application_choice.update!(status: 'recruited', audit_comment: zendesk_url)
+conditions_met = application_choice.offer.conditions.all?(&:met?)
```

Comment:
I wonder if it's worth putting all of this into a method. Typing this by hand or copying and changing variables is prone to error I think.

Reply:
I'd be happy to build up a service for this 👌